docs/articles/documentation/reference/api/runner/reporter.md

```diff
@@ -0,0 +1,89 @@
+---
+layout: docs
+title: Runner.reporter Method
+permalink: /documentation/reference/api/runner/reporter.html
+---
+# Runner.reporter Method
+
+Configures how TestCafe generates test run reports.
+
+```text
+reporter(name, output) → this
+reporter(fn) → this
+reporter([ name | { name, output } | fn ]) → this
+```
+
+Parameter                | Type                        | Description                                     | Default
+------------------------ | --------------------------- | ----------------------------------------------- | --------
+`name`                   | String      | The name of the [reporter](../../../concepts/reporters.md).
+`output`&#160;*(optional)* | String &#124; Writable Stream implementer | The file path where the report is written or the output stream. | `stdout`
+`fn` | A function that [returns a custom reporter object](#specifying-a-custom-reporter).
+
+To use a single reporter, specify a reporter name and, optionally, an output target as the second parameter.
+
+To use multiple reporters, pass an array to this method. This array can include both strings (the reporter name) and `{ name, output }` objects (if you wish to specify the output target). See examples below.
+
+Note that if you use multiple reporters, only one can write to `stdout`.
+
+*Related configuration file property*: [reporter](../../configuration-file.md#reporter)
+
+## Specify the Reporter
+
+```js
+runner.reporter('minimal');
+```
+
+## Save the Report to a File
+
+```js
+runner.reporter('xunit', 'reports/report.xml');
+```
+
+## Use Multiple Reporters
+
+```js
+runner.reporter(['spec', {
+    name: 'json',
+    output: 'reports/report.json'
+}]);
+```
+
+## Specify a Custom Reporter
+
+You can implement a [custom reporter](../../../extending-testcafe/reporter-plugin/README.md) in the code that launches tests. This approach allows you to implement your reporter faster if you need it for a single project or do not want to publish a reporter plugin.
```

Comment:
Custom reporter that launches tests? Or the code that launches tests? Not entirely clear in this case.

Reply:
I'm not sure someone would relate this clause to the reporter tbh.